libs/output-mapping/src/Staging/StrategyFactory.php

```diff
@@ -68,15 +76,25 @@ public function getStrategyMap()
         return $this->strategyMap;
     }
 
-    /**
-     * @param string $stagingType
-     * @return FileStrategyInterface
-     */
-    public function getFileOutputStrategy($stagingType)
+    protected function getStagingDefinition(string $stagingType): Definition
+    {
+        if (!isset($this->getStrategyMap()[$stagingType])) {
+            throw new InvalidInputException(
+                sprintf(
+                    'Input mapping on type "%s" is not supported. Supported types are "%s".',
+                    $stagingType,
+                    implode(', ', array_keys($this->getStrategyMap()))
+                )
+            );
+        }
+        return $this->getStrategyMap()[$stagingType];
```

Comment:
Slo by zkratit na

```
return $this->getStrategyMap()[$stagingType] ?? throw new InvalidInputException(
    sprintf(
        'Input mapping on type "%s" is not supported. Supported types are "%s".',
        $stagingType,
        implode(', ', array_keys($this->getStrategyMap()))
    )
);
```

Reply:
no to je dost nepřehledný, ale aspoň jsem ti to otočil, abys neřekl :)